config/routes.rb

```diff
@@ -1,29 +1,29 @@
 Sidewalks::Application.routes.draw do
 
-  match '/auth/:provider/callback', to: 'sessions#create', via: [ :get, :post ]
-  match '/signin', to: 'sessions#new', as: :signin, via: [ :get, :post ]
-  match '/signout', to: 'sessions#destroy', as: :signout, via: [ :get, :post ]
+  match '/auth/:provider/callback', to: 'sessions#create', via: [:get, :post]
+  match '/signin', to: 'sessions#new', as: :signin, via: [:get, :post]
+  match '/signout', to: 'sessions#destroy', as: :signout, via: [:get, :post]
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.